gh_search.py

```diff
@@ -17,58 +17,52 @@ def search_github(keyword, filetype):
         time.sleep(sleep_time)
         return
     else:
-        print(f'You have {rate.remaining}/{rate.limit} API calls remaining')
+        #print(f'You have {rate.remaining}/{rate.limit} API calls remaining')
         if filetype == 'any':
             query = f'"{keyword}" org:{config.gh_orgname}'
-            result = g.search_code(query, order='desc')
-
-            max_size = 100
-            
-            print(f'Found {result.totalCount} file(s) with {keyword}')
+            searchresults = g.search_code(query, order='desc')
+            # search_code returns a paginated_list https://pygithub.readthedocs.io/en/latest/utilities.html?highlight=pagination#pagination
+            #print(f'Found {searchresults.totalCount} entries with {keyword}')
```

Comment:
Question: should lines 25/26 be log statements rather than comments?

Reply:
Agreed, updated with logging.